src/Module/Profile/Schedule.php

```diff
@@ -33,11 +38,46 @@ public static function content(array $parameters = [])
 			throw new HTTPException\ForbiddenException(DI::l10n()->t('Permission denied.'));
 		}
 
+		if (!empty($parameters['id'])) {
+			self::deleteSchedule($parameters['id']);
```

Comment:
This should be in the `post()` section.

Reply:
The delete is not transmitted as `post`.